src/Adapter/Product/Stock/Update/ProductStockUpdater.php

```diff
@@ -309,5 +316,16 @@ private function saveMovement(StockAvailable $stockAvailable, StockModification
                 'id_shop' => (int) $stockAvailable->id_shop,
             ]
         );
+
+        $this->hookManager->exec(
```

Comment:
Nice catch!! I didn't notice there was a missing hook from the static method, thanks